qa-tests-backend/src/test/groovy/SACTest.groovy

```diff
@@ -98,10 +104,31 @@ class SACTest extends BaseSpecification {
         def img = ImageService.scanImage(TEST_IMAGE, false)
         assert img.hasScan()
 
-        orchestrator.batchCreateDeployments(DEPLOYMENTS)
-        for (Deployment deployment : DEPLOYMENTS) {
-            assert Services.waitForDeployment(deployment)
+        withRetry(30, 5) {
+            for (Deployment deployment : DEPLOYMENTS) {
+                orchestrator.deleteDeployment(deployment)
+            }
```

Comment:
Should this be retried? It might not improve the chances, if we delete the deployments on every retry.